src/hash.cr

```diff
@@ -1343,6 +1343,15 @@ class Hash(K, V)
     self
   end
 
+  # Adds the contents of *other* to this hash. 
+  # The value of each duplicate key is determined by calling the block with the key, its value in this hash and its value in *other*.
+  #
+  #```
+  # hash = { "a" => 100, "b" => 200 }
+  # other = { "b" => 254, "c" => 300 }
+  # hash.merge!(other) { |key, v1, v2| v1 }
```

Comment:
Maybe return `v1 + v2` to show some different behaviour? Could also be an additional example.

Reply:
You're right i chaged it.